src/pages/Chat/Chat.tsx

```diff
@@ -117,7 +147,7 @@ const Chat: React.FC<ChatProps> = ({
     for (let i = 0; i < data.length; i++) {
       let msgObj;
       try {
-        msgObj = JSON.parse(data[i].message);
+        msgObj = data[i].message as any;
```

Comment:
why not parse ?

Reply:
I think with AI filtering on the gateway proxy, something has changed, and parsing will break it.